packages/extension/src/view/popup/components/pieChart.tsx

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies.
+ */
+import React from 'react';
+import { VictoryPie } from 'victory';
+
+interface IPieChartProps {
+  centerCount: number;
+  data: { count: number; color: string }[];
+}
+const PieChart = ({ centerCount, data }: IPieChartProps) => {
+  return (
+    <div className="w-full h-full">
+      <div className="w-full h-full relative">
+        <VictoryPie
+          padding={0}
+          innerRadius={140}
+          animate={{ duration: 400 }}
+          data={data.map(({ count }) => ({ x: '', y: count }))}
+          labels={() => null}
```

Comment:
```suggestion
          labels={() => ''}
```
My text editor shows that `label` is expecting a string or a number.